argocd/metadata.csv

```diff
@@ -0,0 +1,152 @@
+argocd.api_server.process.virtual_memory.bytes,gauge,,byte,,The virtual memory size in bytes in the API Server,0,argocd,api_server process virtual_memory bytes,
+argocd.api_server.process.virtual_memory.max_bytes,gauge,,byte,,The maximum amount of virtual memory available in bytes in the API Server,0,argocd,api_server process virtual_memory max_byte,
+argocd.api_server.redis.request,count,,request,,The number of kubernetes requests executed during application reconciliation in the API Server,0,argocd,api_server redis request,
+argocd.api_server.redis.request.duration.bucket,count,,request,,The Redis requests duration histogram buckets,0,argocd,api_server redis request duration bucket,
+argocd.api_server.redis.request.duration.count,count,,request,,The count aggregation of the Redis requests duration histogram,0,argocd,api_server redis request duration count,
+argocd.api_server.redis.request.duration.sum,count,,request,,The sum aggregation of the Redis requests duration histogram,0,argocd,api_server redis request duration sum,
+argocd.app_controller.app.info,gauge,,,,Information about Applications. It contains tags such as sync_status and health_status that reflect the application state in ArgoCD. The metric value is constant,0,argocd,app_controller app info,
+argocd.app_controller.app.k8s.request,count,,request,,The number of kubernetes requests executed during application reconciliation,0,argocd,app_controller app k8s request,
+argocd.app_controller.app.reconcile.bucket,count,,,,Application reconciliation performance histogram buckets,0,argocd,app_controller app reconcile bucket,
+argocd.app_controller.app.reconcile.count,count,,,,The count aggregation of the application reconciliation performance histogram,0,argocd,app_controller app reconcile count,
+argocd.app_controller.app.reconcile.sum,count,,,,The sum aggregation of the application reconciliation performance histogram,0,argocd,app_controller app reconcile sum,
+argocd.app_controller.app.sync,count,,,,The total number of application syncs,0,argocd,app_controller app sync,
+argocd.app_controller.cluster.api.resource_objects,gauge,,object,,The number of k8s resource objects in the cache,0,argocd,app_controller cluster api resource_objects,
+argocd.app_controller.cluster.api.resources,gauge,,resource,,The number of monitored kubernetes API resources.,0,argocd,app_controller cluster api resources,
+argocd.app_controller.cluster.cache.age.seconds,gauge,,second,,The age of the cluster cache in seconds,0,argocd,app_controller cluster cache age seconds,
+argocd.app_controller.cluster.connection.status,gauge,,,,The current k8s cluster connection status,0,argocd,app_controller cluster connection status,
+argocd.app_controller.cluster.events,count,,event,,The number of processed k8s resource events,0,argocd,app_controller cluster events,
+argocd.app_controller.cluster.info,gauge,,,,,0,argocd,app_controller cluster info,
```

Comment:
It looks like the description is missing from this metric, and its meaning is not clear from its name. Could you confirm and update as needed?